controller/env/appenv.go

```diff
@@ -577,11 +580,9 @@ func NewAuthQueryZitiMfa() *rest_model.AuthQueryDetail {
 }
 
 func NewAuthQueryExtJwt(url string) *rest_model.AuthQueryDetail {
-	provider := rest_model.MfaProvidersURL
 	return &rest_model.AuthQueryDetail{
-		HTTPURL:  url,
-		TypeID:   "EXT-JWT",
-		Provider: &provider,
+		HTTPURL: url,
+		TypeID:  "EXT-JWT",
```

Comment:
"EXT-JWT" really should be a constant imo and it's used in numerous places already

Reply:
This wasn't changed and isn't part of the changes I made here.

This is part of a patch I made to get you off the ground a bit back. I'm not sure why its marking this is changed.